lib/core/versions/latest/models/PublicKeyModel.ts

```diff
@@ -5,5 +5,5 @@ export default interface PublicKeyModel {
   id: string;
   type: string;
   jwk: any;
-  usage: string[];
+  purpose: string[];
```

Comment:
Should the type be `PublicKeyPurpose`?

Reply:
No, this is external facing, thus needs to be identical to what the spec says. I will update line 2 to describe it better!

Reply:
Done.